docs/javascript/browser-apis-and-jquery.md

```diff
@@ -956,3 +956,122 @@ setTimeout(removeEvent, 5000);
 ![offMethod example](https://github.com/codewizardshq/docs/blob/main/docs/assets/browser-apis-and-jquery/offmethod.gif?raw=true){ width="100%" }
 <figcaption></figcaption>
 </figure>
+
+## Getting and Setting attributes
+
+### The `getAttribute()` function allows you to access the value of an element's attribute
+
+```html
+<!DOCTYPE html>
+<html>
+
+<head>
+<title>Home</title>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+
+</head>
+
+<body>
+
+<div>
+  <div class="object" id="one"></div>
+  <img class="object" id="two">
+
+</div>
+<script>
+
+  var elementAttribute = document.querySelector('div').getAttribute("id");
```

Comment:
Let's not chain things if we can help it, as that tends to confuse some kids. Also, it would make more sense to query by ID so it's obvious why "one" appears instead of "two":

```javascript
var elementOne = document.querySelector('#one');
var elementOneID = elementOne.id;
console.log("Element one's ID is: " + elementOneID);
```

Reply:
Sounds good, I changed the example in the next commit.